### PR TITLE
Add section on malware scanning

### PR DESCRIPTION
**Pull request by McPatate**

### CODE REVIEW

File: docs/hub/security.md [hunk 1]
```diff
@@ -165,3 +165,18 @@ git commit -S -m "My first signed commit"
 ```
 
 Once pushed on the Hub, you should see the commit with a "Verified" badge.
+
+## Malware Scanning
+
+We run every file of your repositories through a [malware scanner](https://www.clamav.net/).
```
Comment (julien-c) [inline docs/hub/security.md:171]:
```suggestion
We run every file of your public repositories through a [malware scanner](https://www.clamav.net/).
```

is it only for public repos? i don't remember

File: docs/hub/security.md [hunk 2]
```diff
@@ -165,3 +165,18 @@ git commit -S -m "My first signed commit"
...
+
+Scanning is triggered at each commit or when you visit a repository page.
+
+Here is an example view of an infected file :
+
+![/docs/assets/hub/eicar-hub-tree-view.png](/docs/assets/hub/eicar-hub-tree-view.png)
+
+![/docs/assets/hub/eicar-hub-file-view.png](/docs/assets/hub/eicar-hub-file-view.png)
+
```
Comment (julien-c) [inline docs/hub/security.md:180]: maybe even link to that dummy sample repo here?

File: docs/hub/security.md (hunk 1 above)
Comment (McPatate) [reply docs/hub/security.md:171]: I run it on everything if I'm correct

File: docs/hub/security.md (hunk 2 above)
Comment (McPatate) [reply docs/hub/security.md:180]: Yes why not :)

**Review by julien-c — APPROVED** [review]
LGTM!

You'll need to update the submodule in moon-landing when this is merged, for the doc to appear on the website

File: docs/hub/security.md [hunk 3]
```diff
@@ -165,3 +165,18 @@ git commit -S -m "My first signed commit"
...
+Scanning is triggered at each commit or when you visit a repository page.
+
+Here is an [example view](https://huggingface.co/mcpotato/42-eicar-street/tree/main) of an infected file :
+
+![/docs/assets/hub/eicar-hub-tree-view.png](/docs/assets/hub/eicar-hub-tree-view.png)
+
+![/docs/assets/hub/eicar-hub-file-view.png](/docs/assets/hub/eicar-hub-file-view.png)
+
+_Note_: if your file has neither an ok or infected badge, it could mean that it is either currently being scanned / waiting to be scanned or that there was an error during the scan. It can take up to a few minutes to be scanned.
```
Comment (osanseviero) [inline docs/hub/security.md:181]: Unrelated to this PR, but should we also show a big warning in https://huggingface.co/mcpotato/42-eicar-street when a repo has infected files?

File: docs/hub/security.md [hunk 4]
```diff
@@ -165,3 +165,18 @@ git commit -S -m "My first signed commit"
 ```
 
 Once pushed on the Hub, you should see the commit with a "Verified" badge.
+
+## Malware Scanning
+
+We run every file of your repositories through a [malware scanner](https://www.clamav.net/).
+
+Scanning is triggered at each commit or when you visit a repository page.
```
Comment (osanseviero) [inline docs/hub/security.md:173]: Do we trigger the scanning every time any user goes to a repo?

Comment (McPatate) [reply docs/hub/security.md:173]: Yes, if it hasn't been scanned already (for a given revision)

File: docs/hub/security.md (hunk 3 above)
Comment (McPatate) [reply docs/hub/security.md:181]: It's kind of there already, just on the files tab

Comment (osanseviero) [reply docs/hub/security.md:181]: I don't think it's the same. As a user of a model, I would likely not open the files tab, just read the model card and even click the "how to clone" button in the model card to download the model. If a model is scanned to be unsafe, we should show it more prominently imo. (happy to follow up discussing this in slack :D)

**Comment by julien-c** [comment]
Ok for me to merge this! (and deploy on hf.co)

File: docs/hub/security.md (hunk 3 above)
Comment (McPatate) [reply docs/hub/security.md:181]: I've just made it so that calls to my API take on avg 5-10 ms. Is that too much of an overhead ?

Comment (julien-c) [reply docs/hub/security.md:181]: no 5-10ms is fine

**Pull request merged**